Lib/tkinter/dnd.py

```diff
@@ -99,8 +99,9 @@
 
 """
 
+__all__ = ["dnd_start", "DndHandler"]
 
-import tkinter
+from tkinter import *
```

Comment:
Why is this change?

Reply:
I wanted to make it more similar to the other files. But I get your point: keep changes minimal, this is indeed just for style